src/hint_processor/builtin_hint_processor/builtin_hint_processor_definition.rs

```diff
@@ -249,7 +249,7 @@ impl HintProcessor for BuiltinHintProcessor {
             hint_code::BLAKE2S_COMPUTE => {
                 compute_blake2s(vm, &hint_data.ids_data, &hint_data.ap_tracking)
             }
-            hint_code::VERIFY_ZERO_V1 | hint_code::VERIFY_ZERO_V2 => {
+            hint_code::VERIFY_ZERO_V1 | hint_code::VERIFY_ZERO_V2 | hint_code::VERIFY_ZERO_V3 => {
```

Comment:
Actually, here we shouldnt execute the same functions.
It is just a minor detail but see that this hint you are implementing doesn't import the SECP constant, so in this cases, we should get the SECP constant from the scope

Reply:
Done!